yodeploy/cmds/yodeploy_server.py

```diff
@@ -38,6 +38,7 @@ def load_config(self):
 
         self.cfg.set('bind', '{}:{}'.format(opts.listen, opts.port))
         self.cfg.set('ciphers', config.server.ssl.ciphers)
+        self.cfg.set('ssl_version', config.server.ssl.tls)
```

Comment:
This selects a specific version of the protocol, rather than letting auto-negotiation occur (which could select 1.3, for example).

Generally [OP_NO_TLSv1](https://docs.python.org/2/library/ssl.html#ssl.OP_NO_TLSv1) is the better way to handle blocking old protocols.

Reply:
Gunicorn doesn't seem to support that yet, https://github.com/benoitc/gunicorn/pull/1890 adds this in 20.0 which isn't released yet, lets keep specific version till that lands and then switch it? I'll create issue to track that ofc

Reply:
That one doesn't help, but benoitc/gunicorn#1140 will.

Reply:
Name it `ssl_version` for consistency?